### ISSUE TITLE
Terminal does not continue running when switching to other apps

### ISSUE THREAD

**Issue by GoogleCodeExporter** [globelblade/mobileterminal#28]
```
1. Open Terminal, and type something
2. Click the 'home' button
3. Open Terminal again.  A fresh shell will be running.

It would be very nice to start terminal and see the previous session.  
Currently responding to an SMS or receiving a phone call will 
terminate what you are doing.  That is very bad.

One possible (but kinda hacky) solution is to add the ability to configure a 
command to be run when terminal starts, and have it 
run a script to launch 'screen' with the proper arguments.  Effectively this 
would have the same result as keeping the application 
running the whole time.  The biggest problem with this however is that it 
basically means that you can not use screen inside the 
terminal. The upshot though is that even if Terminal.app crashes the session 
will keep running.

Perhaps a modified version of screen could be included with Terminal.app that 
uses different key strokes, and give Terminal.app 
buttons or menu items to use those key strokes.  This would easily give 
features such as multiple tabs, and such, and still allow 
running screen within a session because that would use the original screen 
keystrokes.
```

Original issue reported on code.google.com by `britr...@gmail.com` on 5 Aug 2007 at 3:30
